src/main/java/core/basesyntax/calculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface calculateArea {
+    double getArea();

```

Comment:
Rename it to `calculateArea` since you make calculations, not just getting value from some field.